sdk/dotnet/Pulumi/Core/Output.cs

```diff
@@ -101,8 +101,11 @@ public static Output<string> Format(FormattableString formattableString)
                 string.Format(formattableString.Format, objs.ToArray()));
         }
 
+        internal static Output<ImmutableArray<T>> Concat<T>(Output<ImmutableArray<T>> values, Output<T> value)
```

Comment:
Again I would think this would be better named Append

Reply:
Happy to rename things if that helps.

Reply:
This can actually be removed with the params signature restored since it uses the other overload.